submissions/sprint_a/app.py

```diff
@@ -0,0 +1,31 @@
+from flask import jsonify
+
+from sprint_a import app
+from sprint_a.helpers import EpithetGenerator
+from sprint_a.helpers import Vocabulary
+
+path = ('/Users/aj/Documents/Back-End/backend-epithet-generator/resources/data.json')
```

Comment:
Use relative paths or absolute paths built by joining PROJECT_ROOT declared in init file.